src/components/breakpointHandler.ts

```diff
@@ -0,0 +1,667 @@
+import * as vscode from 'vscode';
+import { DaCeVSCode } from '../extension';
+import { SdfgViewerProvider } from "./sdfgViewer"
+import fs = require("fs");
+
+export class Node {
+    sdfg_id: number;
+    state_id: number;
+    node_id: number;
+
+    cache: string | undefined;
+    target: string | undefined;
+    sdfg_name: string | undefined;
+
+    constructor(sdfg_id: number, state_id: number, node_id: number) {
+        this.sdfg_id = sdfg_id;
+        this.state_id = state_id;
+        this.node_id = node_id;
+    }
+
+    public printer(): string {
+        return this.sdfg_id + ':' +
+            this.state_id + ':' +
+            this.node_id;
+    }
+
+    public isEqual(node: Node): boolean {
+        return this.sdfg_id == node.sdfg_id &&
```

Comment:
Similar to the semicolons, please try to use the strict equality operator (`===`) wherever possible, and `==` only when necessary